src/pages/whatsnew/2.46.1/en.mdx

```diff
@@ -0,0 +1,11 @@
+import { WhatsNewLayout } from '../../../components/WhatsNewLayout';
+
+### You can now mention user in comment!
+
+All mentioned users will receive email.
+
+![](/MentionDropdown.png)
+
+_Use it and test. Leave feedback at any time via button in footer from any page._
```

Comment:
Try it now! Your your feedback is highly appreciated, just click "Feedback" on the bottom of any page.